### PR TITLE
docs: readme update

### PR DESCRIPTION
**Pull request by esciara**
Properly documented readme file.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,223 @@
+======================
+Cookiecutter PyPackage
```
Comment (antoinemichaud) [inline README.rst]: Change the title : cookiecutter doesn't matter. May be some day we will change this engine for another ? Focus on the target - having an opinionated full python library project.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+======================
+
+.. image:: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/badges/master/pipeline.svg
+     :target: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/pipelines
+
+
+Cookiecutter_ template for a Python package.
```
Comment (antoinemichaud) [inline README.rst]: Same comment

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+    # Minimal setup
+
+    $ make setup-dev-env-minimal
+
+    # OR alternatively, full setup (allows for IDE completion)
+
+    $ make setup-dev-env-full
+
+* Create a repo on Github or Gitlab (cloud or hosted) and put it there.
```
Comment (antoinemichaud) [inline README.rst]: "Put it there" is ambiguous : what ? and where ?

**Review by antoinemichaud — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (esciara) [reply README.rst]: Yes but on the other hand, the project name is `cookiecutter-pypackage`. Do you suggest to change the whole thing? Can this wait for future releases?

File: README.rst (hunk 2 above)
Comment (esciara) [reply README.rst]: same answer 😬

File: README.rst (hunk 3 above)
Comment (esciara) [reply README.rst]: Made changes, please review

File: README.rst (hunk 1 above)
Comment (esciara) [reply README.rst]: Following exchange, will replace with `Opinionated Python Library Project Template`.

File: README.rst (hunk 2 above)
Comment (esciara) [reply README.rst]: ~Following exchange, will replace with `Opinionated Python Library Project Template`~ Actually, that would be a repeat of the title. Simply skipping it.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,231 @@
+======================
+Cookiecutter PyPackage
+======================
+
+.. image:: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/badges/master/pipeline.svg
+     :target: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/pipelines
```
Comment (esciara) [inline README.rst]: Precise that this is on linux

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+Linux:
+
+.. image:: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/badges/master/pipeline.svg
+    :target: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/pipelines
+    :alt: Linux build status on Gitlab CI
+
+Windows:
+
+This template is not tested on Windows yet, and some feature will most probably not be
+working on it, unless you use `Windows Subsystem for Linux`_.
```
Comment (esciara) [inline README.rst]: Remove

**Comment by esciara** [comment]
reviewed with @slequeux and @antoinemichaud

**Pull request merged**